<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputerForm</class>
 <widget class="QWidget" name="AddComputerForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonAddComputer">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Computer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComputerType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Computer Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYearBuilt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Year Built</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComputerName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Computer Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxWasBuilt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Was Built</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
